src/ontology/uberon-edit.obo

```diff
@@ -223958,6 +223958,66 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: part_of UBERON:0000916 ! abdomen
 property_value: dcterms-date "2023-04-26T07:54:47Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480054
+name: ileocecal lymph node
+def: "Mesenteric lymph node embedded in the ileocecal fold at both sides of the concavity of the cecum. It drains the terminal ileum." [PMID:15493265, PMID:18251298]
+synonym: "ileocaecal lymph node" EXACT [PMID:15493265]
+is_a: UBERON:0002509 ! mesenteric lymph node
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+property_value: curator_notes "According to the Segen's Medical Dictionary, Farlex, Inc., 2011 the ileocaecal lymph node is an obsolete term that formerly dignified those ileocolic lymph nodes lying in the ileocaecal angle (https://medical-dictionary.thefreedictionary.com/ileocaecal+lymph+nodes). However, this term is still used in domesticated animals as pigs (e.g. PMID:31031713), cats (e.g. PMID:35964104), sheeps (e.g. PMID:15493265) and calves (e.g. PMID:23000583). In few cases this term has been used in humans (PMID:32420296, PMID:35277366, PMID:31456616). Consider if appropriate as a subclass of ileocolic lymph node (UBERON:0016378)." xsd:string
+property_value: dcterms-date "2023-05-11T12:09:21Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480055
+name: subcarinal lymph node
+def: "Superior mediastinal lymph node located in the thoracic cavity between the lungs. It is bordered by the carina of the trachea, the lower lobe bronchus on the left and the intermediate bronchus on the right." [NCIT:C117873, PMID:30422458]
+synonym: "left subcarinal lymph node" NARROW [PMID:36899970]
+synonym: "subcarinal node" EXACT [PMID:30422458]
+is_a: UBERON:0038651 ! superior mediastinal lymph node
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+property_value: curator_notes "Subcarinal lymph nodes (station 7) in International Association for the Study of Lung Cancer (IASLC) map coincide with station 2, 8, and 10 lymph nodes in Wang’s map. PMID:26807271" xsd:string
+property_value: dcterms-date "2023-05-11T12:12:47Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480056
+name: left supraclavicular lymph node
+def: "Supraclavicular lymph node located on the left side of a entire appendicular skeleton. The left supraclavicular lymph node drains the kidney, cervix, testis, and pancreas into the thoracic duct." [PMID:31335020]
+is_a: UBERON:0035279 ! supraclavicular lymph node
```

Comment:
These should have logical definitions as they follow the laterality pattern. But can be done in a separate PR